app/controllers/v1/people_controller.rb

```diff
@@ -1,7 +1,7 @@
 class V1::PeopleController < V1::BaseController
 
   def create
-    @person = PersonWithRemoteFields.find_or_build(person_params)
+    @person = PersonConstructor.new(person_params).build
```

Comment:
I don't feel as strongly and would have my constructor return the object it is building.  Regardless, 
I'd try to avoid having coupled methods that always act in tandem.  Instead of having build as an instance method, I'd set it up as a class method.  Then replace your `PersonConstructor.new(person_params).build` with ` PersonConstructor.build(person_params)`